portfolio/src/main/java/com/google/sps/authentication/userservice/UserServiceInteraction.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.authentication.userservice;
+
+import com.google.appengine.api.users.UserService;
+
+/** Interface that handles the interaction with User Service API. */
```

Comment:
Interface maybe confused with java's interface? Maybe what you mean is a wrapper for user service? If so, could rename to UserServiceWrapper? up to you.

Reply:
My god forgot to rephrase it... fixed

Reply:
my god forgot to rephrase... fixed